### PR TITLE
enterprise: fix API mixin license validity check

### PR DESCRIPTION
**Pull request by rissson**
## Details

REPLACE ME

---

## Checklist

-   [ ] Local tests pass (`ak test authentik/`)
-   [ ] The code has been formatted (`make lint-fix`)

If an API change has been made

-   [ ] The API schema has been updated (`make gen-build`)

If changes to the frontend have been made

-   [ ] The code has been formatted (`make web`)

If applicable

-   [ ] The documentation has been updated
-   [ ] The documentation has been formatted (`make website`)

### CODE REVIEW

File: authentik/enterprise/api.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ class EnterpriseRequiredMixin:
 
     def validate(self, attrs: dict) -> dict:
         """Check that a valid license exists"""
-        if LicenseKey.cached_summary().status != LicenseUsageStatus.UNLICENSED:
+        if LicenseKey.cached_summary().status != LicenseUsageStatus.VALID:
```
Comment (gergosimonyi) [inline authentik/enterprise/api.py:32]: I don't have any context, but

```suggestion
        if LicenseKey.cached_summary().status.is_valid:
```

?

Comment (rissson) [reply authentik/enterprise/api.py]: Yep, that looks like a better check

**Review by gergosimonyi — APPROVED** [review]

**Review by Jeremiegmoore — COMMENTED** [review]
[``](url)

**Comment by Jeremiegmoore** [comment]
> [``](url)

**Pull request merged**